packages/storybook-components/src/ReadMeHidePreview.js

```diff
@@ -15,11 +15,11 @@ const PreviewComponent = ({ children }) =>
     </div>
   );
 
-const ReadMeHidePreview = (readme, ...rest) =>
-  withDocs({ PreviewComponent })(navigator.userAgent.match(/Chromatic/) ? '' : readme, ...rest);
+const readMeHidePreview = (readme = '') =>
```

Comment:
we should probably change the filename as well - probably to kebab-case

Reply:
I opened an issue for this #633